modules/ROOT/pages/documentation-openapi.adoc

```diff
@@ -129,15 +129,16 @@ For more information, see https://openliberty.io/guides/microprofile-openapi.htm
 
 == Filter components in and out of API documentation
 
+If you want to update or remove certain elements and fields of the OpenAPI document you can use a filter, which is called once after all other documentation mechanisms have completed. .
```

Comment:
_which is called once after_

this reads strangely- is _once_ a typo or does it mean the filter is called one time.

Reply:
Once likely means one time. Will check with Martin.